### PR TITLE
[ZEPPELIN-3007] display a note name without any path in the tab title

### PR DESCRIPTION
**Pull request by tinkoff-dwh**
### What is this PR for?
This PR modifies the display of the name of the note.
Now, instead of the full path, only the name of the note is displayed.

### What type of PR is it?
Improvement

### What is the Jira issue?
[ZEPPELIN-3007](https://issues.apache.org/jira/browse/ZEPPELIN-3007)

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: zeppelin-web/src/app/app.controller.js [hunk 1]
```diff
@@ -46,7 +46,9 @@ function MainCtrl ($scope, $rootScope, $window, arrayOrderingSrv) {
 
   $rootScope.noteName = function (note) {
     if (!_.isEmpty(note)) {
-      return arrayOrderingSrv.getNoteName(note)
+      let name = arrayOrderingSrv.getNoteName(note)
+      name = name.substr(name.lastIndexOf('/') + 1, name.length)
```
Comment (felixcheung) [inline zeppelin-web/src/app/app.controller.js]: is this going to work on Windows?

Comment (tinkoff-dwh) [reply zeppelin-web/src/app/app.controller.js]: Yes, it works on Windows

**Comment by tinkoff-dwh** [comment]
default

**Comment by Leemoonsoo** [comment]
Tested this branch.

zeppelin_notename

Things little bit inconsistent to me is when renaming note name although i don't have any good idea on it.
@tinkoff-dwh @felixcheung what do you guys think?

**Comment by tinkoff-dwh** [comment]
I think it's better to show a pop-up when clicking on the name of the note. As on the main page.
 
capture2

What do you think about it?

**Comment by tinkoff-dwh** [comment]
@Leemoonsoo 
What about this option?

capture5
Now note name clipped on the left.

**Comment by Leemoonsoo** [comment]
@tinkoff-dwh Looks good to me!

Merge to master if no further comment.

**Pull request closed without merging**